### PR TITLE
Instant Search: Add the "Clear filters" button to the tab loop.

### PR DESCRIPTION
**Pull request by a8ck3n**
Just a standard looking button at the moment but it works across all browsers now and doesn’t break tabbing.

Addresses part of #19035.

#### Changes proposed in this Pull Request:

The "Clear filters" button is now a `button` instead of an `a`. This allows it to participate in the tab loop in Firefox and no longer breaks the loop in other browsers.

**Note:** Just a standard button now. The old styling wasn't that nice but we can tweak that based on feedback.

Before:
ClearOld

After:
ClearNew

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

1. Open the Instant Search overlay.
2. Click on a filter or two so the "Clear filters" button appears.
3. Try tabbing to it.
4. Confirm that both Return and Space activate the button and clear the filters.

### CODE REVIEW

**Review by kangzj — COMMENTED** [review]
It works as described. However do you think it's better to keep the styling the same as it was ? It had been a button before it was changed to a link in #14762 where @keoshi provided design review.

**Comment by a8ck3n** [comment]
> It works as described. However do you think it's better to keep the styling the same as it was? It had been a button before it was changed to a link […] provided design review.

Thanks for flagging those previous changes. I've updated the styling to match so it's once again looking like a link. Re-tested in the big 3 browsers and it looks pretty good.

File: projects/packages/search/src/instant-search/components/search-filters.scss [hunk 1]
```diff
@@ -24,6 +24,13 @@
 	top: 0;
 	margin: 0;
 	line-height: 1.3;
+	border: none;
+	background-color: inherit;
+
+	&:hover,
+	&:focus {
+		text-decoration: underline;
```
Comment (dognose24) [inline projects/packages/search/src/instant-search/components/search-filters.scss]: I would suggest leaving text-decoration: none is appropriate to keep styles the same between the `sort` buttons and the `Clear filters` button for `:hover` and `:focus`.

截圖 2022-08-03 上午3 26 13
截圖 2022-08-03 上午3 32 17

Comment (a8ck3n) [reply projects/packages/search/src/instant-search/components/search-filters.scss]: Ah, nice catch. Thanks @dognose24. I'll get that fixed up.

**Comment by a8ck3n** [comment]
Thanks for the review @dognose24. I've updated the styles there to remove the `text-decoration` and add the `cursor` attribute. The two button styles are a little more consistent now. Would you mind giving this another quick look?

**Review by dognose24 — APPROVED** [review]
The sort and filter clearing button styles look consistent when focusing and hovering on browsers Safari, Firefox, and Chrome!

**Pull request merged**